### ISSUE TITLE
Visual lockscreen bug

### ISSUE THREAD

**Issue by brog-io** [ente-io/ente#3370]
### Description

When opening the app the lockscreen bugs a little bit. This doesn't appear to happen on Ente Photos. 

https://github.com/user-attachments/assets/f5806472-bd8e-45d1-92bb-42fa2ae78d55

### Version

Latest

### What product are you using?

Ente Auth

### What platform are you using?

Mobile - Android

**Comment by ashilkn** [comment]
Does this happen only when opening the app after searching for it?
What I'm suspecting is, if the keyboard is open just before the app is opened, this bug comes up.

**Comment by brog-io** [comment]
> Does this happen only when opening the app after searching for it?
> What I'm suspecting is, if the keyboard is open just before the app is opened, this bug comes up.

Yes, it only happens when I freshly open the app. If it's open in the background this doesn't happen.

**Comment by Gururagavendra** [comment]
I stil dont get the bug, can u please explain it @brog-io

**Comment by Gururagavendra** [comment]
is the keyboard popping at the beginning is the issue?

**Comment by brog-io** [comment]
> is the keyboard popping at the beginning is the issue?

Yes. Its very minor. But just a little visual annoyance

**Comment by AmanRajSinghMourya** [comment]
Hey @brog-io could you please check if the issue still exists.

**Comment by brog-io** [comment]
> Hey @brog-io could you please check if the issue still exists. 

Just tested on 4.4.15 and it still exists

**Comment by brog-io** [comment]
> Does this happen only when opening the app after searching for it?
> What I'm suspecting is, if the keyboard is open just before the app is opened, this bug comes up.

This is almost definitely the cause, it only happens when I search for the app and open it. When using the new shortcut feature it doesnt happen.

**Comment by ua741** [comment]
Is this still happening on the latest version cc @brog-io ?

**Comment by brog-io** [comment]
> Is this still happening on the latest version cc @brog-io ? 

Yes, just replicated it
https://github.com/user-attachments/assets/9bb08205-dcf0-4cda-8507-925ae0be0f27

**Comment by AmanRajSinghMourya** [comment]
@brog-io Could you please retest this on Auth v4.4.25? Force-stop Auth first, then open it from launcher search while the keyboard is visible. We can no longer reproduce it. If it persists, please share new recording.
